PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/utility/AccessPrivate.java

```diff
@@ -1,4 +1,4 @@
-package nl.tudelft.pl2016gr2.test.utility;
```

Comment:
I changed it because I found it very confusing that a package containing 'test' is not in src/test but in src/main. Checkstyle had difficulties with that class so I tried to find it which was confusing because I was first looking in src/test. Maybe we should move the utility package to src/test.